spring-cloud-cli/src/test/java/org/springframework/cloud/cli/command/url/UrlEncodeCommandTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2013-2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.cli.command.url;
+
+import org.junit.Test;
+import org.springframework.boot.cli.command.Command;
+import org.springframework.boot.cli.command.status.ExitStatus;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author William Witt
+ */
+public class UrlEncodeCommandTest {
+    Command command = new UrlEncodeCommand();
+
+    @Test
+    public void urlEncode() throws Exception{
+        assertEquals(ExitStatus.OK, command.run("abcdefg"));
+        assertEquals(ExitStatus.OK, command.run("a b c&d%efg+"));
```

Comment:
I would make sure that the printed value is correct. There is an `OutputCapture` rule you can use.